iplayer.el

```diff
@@ -246,15 +251,23 @@ The presets are defined in the variable `iplayer-presets'."
   (catch 'no-progress
     (cond
      ((or (string-match "^Starting download" string)
-          (string-match "^INFO: Begin recording" string))
+          (string-match "^INFO: Begin recording" string)
+          (string-match "INFO: Downloading tv" string))
```

Comment:
So, it's been a while since I've used this -- that means that my comments should be taken with a pinch of salt.

This new string-match line mentions `tv` explicitly, which makes me wonder whether it would work for radio downloads.  Have you verified that?